src/main/java/roomescape/reservation/repository/JdbcReservationRepository.java

```diff
@@ -0,0 +1,148 @@
+package roomescape.reservation.repository;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import javax.sql.DataSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.reservation.domain.Reservation;
+import roomescape.reservationtime.domain.ReservationTime;
+import roomescape.theme.domain.Theme;
+
+@Repository
+public class JdbcReservationRepository implements ReservationRepository {
+
+    private static final RowMapper<Reservation> reservationRowMapper = (resultSet, rowNum) -> {
+        final DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+        final LocalTime time = LocalTime.parse(resultSet.getString("time_value"), timeFormatter);
+        return new Reservation(
+                resultSet.getLong("id"),
+                resultSet.getString("name"),
+                resultSet.getObject("date", LocalDate.class),
+                new ReservationTime(resultSet.getLong("time_id"), time),
+                new Theme(resultSet.getLong("theme_id"), resultSet.getString("theme_name"),
+                        resultSet.getString("theme_description"), resultSet.getString("theme_thumbnail"))
+        );
+    };
+
+    private final JdbcTemplate template;
+    private final SimpleJdbcInsert inserter;
```

Comment:
변수명을 축약해서 표현하신 이유가 있을까요?

```suggestion
    private final JdbcTemplate jdbcTemplate;
    private final SimpleJdbcInsert simpleJdbcInsert;
```

예를들면

> template.queryForObject(sql, Integer.class, date, time, themeId);

해당 라인만 보면 무엇을 사용하는지 확신보다 추측을 먼저 할 것 같아요
그 뒤에 필드를 확인하러 가서 확신을 가지게 되구요

Reply:
이유는 없었습니다. 
그냥 관성적으로 썼던 것 같습니다. 
찰리 말대로 축약보다 명확하게 표현하는게 더 좋아보입니다.  
감사합니다.